app/connectors/BusinessRatesAttachmentConnector.scala

```diff
@@ -0,0 +1,74 @@
+/*
+ * Copyright 2019 HM Revenue & Customs
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package connectors
+
+import javax.inject.{Inject, Singleton}
+
+import config.WSHttp
+import models.attachment.InitiateAttachmentRequest
+import models.attachment.SubmissionTypesValues.PropertyLinkEvidence
+import models.upscan.PreparedUpload
+import play.api.Logger
+import play.api.libs.json.{JsObject, JsString, Json}
+import uk.gov.hmrc.http.{HeaderCarrier, HttpErrorFunctions}
+import models.attachment._
+import uk.gov.hmrc.http._
+import uk.gov.hmrc.play.config.ServicesConfig
+
+import scala.concurrent.{ExecutionContext, Future}
+import scala.concurrent.ExecutionContext.Implicits.global
+
+@Singleton
+class BusinessRatesAttachmentConnector @Inject()(val  http: WSHttp)(implicit val appConfig: ServicesConfig, executionContext: ExecutionContext)
```

Comment:
why is servicesConfig coming in implicitly?

Reply:
Now servicesConfig injecting